changes/v5.0.20-en.md

```diff
@@ -0,0 +1,5 @@
+# v5.0.20
```

Comment:
The way we generate changelogs has changed recently.  Now we create one markdown file per {fix,feat}x{en,md} under `changes/{ce,ee}/`, referencing the PR number only in the filename.

So, for this PR, it would go to `changes/ce/fix-10098.{en,zh}.md` with the contents like:

```
When configuring mongodb authorization the mongodb connector crashed with an error in the log file. The reason was that the collection name had the wrong type.
```

Reply:
ahh, yeah, of course. I don't understand how I could forget.

Reply:
Fixed